src/main/java/io/appium/java_client/service/local/AppiumDriverLocalService.java

```diff
@@ -44,7 +44,7 @@
     private final String ipAddress;
     private final long startupTimeout;
     private final TimeUnit timeUnit;
-    private final ReentrantLock lock = new ReentrantLock();
+    private final ReentrantLock lock = new ReentrantLock(true);
```

Comment:
Maybe add a comment here mentioning true means `fair ordering policy`? It's not obvious what `true` does in the absence of a comment.